pages/common/gcloud.md

```diff
@@ -0,0 +1,32 @@
+# gcloud
+
+> The official CLI tool for Google Cloud Platform. You can use this tool to perform many common platform tasks either from the command-line or in scripts and other automations.
+> Commands can be run using the _gcloud_ command.
+
+- List all properties in your active configuration. Including: current authorized account, current project, and the default region and zone, if set.:
+
+`gcloud config list`
+
+- Set the active project:
+
+`gcloud config set project {{project_name}}`
+
+- SSH into a virtual machine instance:
+
+`gcloud compute ssh {{user}}@{{instance}} `
+
+- Display all Google Compute Engine instances in a project. Instances from all zones are listed as default, results can be narrowed down using a filter, eg: --filter="zone:(us-central1-a)":
```

Comment:
Perhaps split the filter example into another example if it's commonly used?

Reply:
SGTM